testsuite/MDAnalysisTests/test_util.py

```diff
@@ -531,22 +531,23 @@ class TestFixedwidthBins(object):
     def test_keys(self):
         ret = util.fixedwidth_bins(0.5, 1.0, 2.0)
         for k in ['Nbins', 'delta', 'min', 'max']:
-            assert k in ret
+            if k not in ret:
+                raise Exception
```

Comment:
`AssertError`, same as above. Yet, here, the code may be clearer is you use `assert_` .